src/shared/views/ResourceView.tsx

```diff
@@ -1,367 +1,40 @@
-                    onSubmit={handleEditFormSubmit}
-                    onExpanded={handleExpanded}
-                  />
-                </TabPane>
-                <TabPane tab="History" key="#history">
-                  <HistoryContainer
-                    resourceId={resource['@id']}
-                    orgLabel={orgLabel}
-                    projectLabel={projectLabel}
-                    latestRev={latestResource._rev}
-                    link={(rev: number) => {
-                      return (
-                        <a
-                          onClick={() => {
-                            goToResource(orgLabel, projectLabel, resourceId, {
-                              revision: rev,
-                            });
-                          }}
-                        >
-                          Revision {rev}
-                        </a>
-                      );
-                    }}
-                  />
-                </TabPane>
-                <TabPane tab="Links" key="#links">
-                  <div
-                    style={{ display: 'flex', justifyContent: 'space-between' }}
-                  >
-                    <div style={{ width: '48%' }}>
-                      <h3 style={{ paddingLeft: '7px' }}>Incoming</h3>
-                      <ResourceLinksContainer
-                        resourceId={resource['@id']}
-                        orgLabel={orgLabel}
-                        projectLabel={projectLabel}
-                        rev={resource._rev}
-                        direction="incoming"
-                        onClick={handleGoToInternalLink}
-                      />
-                    </div>
-                    <div style={{ width: '48%' }}>
-                      <h3 style={{ paddingLeft: '7px' }}>Outgoing</h3>
-                      <ResourceLinksContainer
-                        resourceId={resource['@id']}
-                        orgLabel={orgLabel}
-                        projectLabel={projectLabel}
-                        rev={resource._rev}
-                        direction="outgoing"
-                        onClick={handleGoToInternalLink}
-                      />
-                    </div>
-                  </div>
-                </TabPane>
-              </Tabs>
-            </>
-          )}
-        </Spin>
-      </div>
-      <div
-        ref={ref}
-        className="graph-wrapper"
-        style={{
-          width: '100%',
-          height: '100%',
-          position: 'relative',
-        }}
-      >
-        <div
-          style={{
-            width,
-            position: 'fixed',
-            height: '100%',
-            maxHeight: 'calc(100vh - 40px)',
-          }}
-        >
-          {resource && <GraphContainer resource={resource} />}
-        </div>
-      </div>
+      <ResourceViewContainer
+        render={resource => (
+          <div
+            ref={ref}
+            className="graph-wrapper"
+            style={{
```

Comment:
Nit Pick: Do we need these inline styles here? Can we move them to graph-wrapper to create or create a new class?